app/viewmodels/CashAccountDailyStatementsViewModel.scala

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels
+
+import models.{CashDailyStatement, CashTransactionType, CashTransactions, Declaration, Payment, Transaction, Transfer, Withdrawal}
```

Comment:
probably worth just doing models._ this is quite long.

Reply:
Its fine here, as these are only 8 models.  If we use * here, more than twice of the current models will be imported unnecessarily and will increase the compile time.  Also its better to be specific in the imports.

Reply:
We can discuss this further in person during Leeds visit. Resolving as PR has been approved